sdk/mosaic_model.go

```diff
@@ -266,3 +268,39 @@ func Streaming(amount uint64) *Mosaic {
 func SuperContractMosaic(amount uint64) *Mosaic {
 	return newMosaicPanic(SuperContractNamespaceId, Amount(amount))
 }
+
+/// region Levy information
+type MosaicLevy struct {
+	Type 		uint16
```

Comment:
I noticed that some of the lines or text are misaligned. 

![image](https://user-images.githubusercontent.com/4479171/87228960-96170300-c372-11ea-844a-5449941b8392.png)